test/test_experiment_fragment.py

```diff
@@ -51,6 +52,12 @@ def test_recursive_rebind_override(self):
         result = run_fragment_once(rrf)[rrf.rebound_add_one.add_one.result]
         self.assertEqual(result, 3)
 
+    def test_multiple_rebinds(self):
+        mrb = self.create(MultipleRebindsFragment, [])
+        results = run_fragment_once(mrb)
+        for fragment in mrb.fragments:
+            self.assertEqual(results[fragment.result], 1)
```

Comment:
See comment above - if changing the rebound default to `1`, this should be `2` in all subfragments.